R/calculate.R

```diff
@@ -17,11 +17,14 @@
 #' @param start_year Numeric, holding start year of analysis.
 #' @param end_year Numeric, holding end year of analysis.
 #' @param time_horizon Considered timeframe for PACTA analysis.
+#' @param growth_rate Numeric, that holds the terminal growth rate of profits
+#'   beyond the final year in the DCF.
```

Comment:
please instead of final year mention `end_year` so it is obvious how args are related

Reply:
done

Reply:
done